pwndbg/commands/context.py

```diff
@@ -1088,17 +1088,15 @@ def context_backtrace(with_banner=True, target=sys.stdout, width=None):
 
     frame = newest_frame
     i = 0
-    bt_prefix = "%s" % pwndbg.config.backtrace_prefix
+    bt_prefix = f"{pwndbg.config.backtrace_prefix}"
     while True:
         prefix = bt_prefix if frame == this_frame else " " * len(bt_prefix)
         prefix = f" {c.prefix(prefix)}"
         addrsz = c.address(pwndbg.ui.addrsz(frame.pc()))
         symbol = c.symbol(pwndbg.aglib.symbol.resolve_addr(int(frame.pc())))
         if symbol:
-            addrsz = addrsz + " " + symbol
-        line = map(str, (prefix, c.frame_label("%s%i" % (backtrace_frame_label, i)), addrsz))
-        line = " ".join(line)
-        result.append(line)
+            addrsz = f"{addrsz} {symbol}"
+        result.append(f"{prefix} {c.frame_label(f'{backtrace_frame_label}{i}')} {addrsz}")
```

Comment:
please check if this works fine in python3.10

Reply:
ok

Reply:
```
$ docker run --rm -it python:3.10 python
Unable to find image 'python:3.10' locally
3.10: Pulling from library/python
a492eee5e559: Pull complete
32b550be6cb6: Pull complete
35af2a7690f2: Pull complete
7576b00d9bb1: Pull complete
ff1cff00b1dd: Pull complete
57e75b8db17c: Pull complete
e844e4b3e270: Pull complete
Digest: sha256:76f22e4ce53774c1f5eb0ba145edb57b908e7aa329fee75eca69b511c1d0cd8a
Status: Downloaded newer image for python:3.10
Python 3.10.16 (main, Feb  4 2025, 07:28:23) [GCC 12.2.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> f'a {f"{123}"} b'
'a 123 b'
```

Reply:
```
>>> def f(x): return 'x'+str(x)+'x'
...
>>> f'a {f"{f(123)}"} b'
'a x123x b'
```